### DISCUSSION TITLE
expansion of electrons when there is no laser

### DISCUSSION THREAD

**Discussion by dpkhatua** [epochpic/epoch#514] [General]
Hii all,
I tried to simulate one code for Au bar initialized from single ionized state at room temperature and without any laser source. From the plot of electric field, I found that the electrons are expanding with a speed of light generating a high electric field of the order of 10^9 V/m. To smooth the current, I use the smooth_current = T statement. still, the result is same. 
Can anyone please suggest me what may be the reason and how to get rid of it?

![E6](https://github.com/Warwick-Plasma/epoch/assets/63596383/3db5f342-b220-45b3-9997-d70976b14f3d)

Thanks.

**Comment by Status-Mirror** [comment]
For a solid density target at room temperature (~0.02 eV), you will experience significant self-heating. I believe this is what causes your electrons to reach high speeds.

You cannot remove self-heating from PIC codes, but you can slow down the self-heating rate by using larger macro-particle shapes, smaller cells and more particles per cell. However, I don't think you would reach the resolution required to model a room-temperature solid-density plasma. Most people who study high intensity laser-solid interactions would set an initial temperature to around 1-5 keV, where self-heating is reduced.

Hope this helps,
Stuart
